content/en/docs/Overview/_index.md

```diff
@@ -66,22 +66,20 @@ Besides its own Web UI, Navidrome should be compatible with all Subsonic clients
 - Connected Speakers:
   - Sonos: [bonob](https://github.com/simojenki/bonob#readme)
   - Alexa: [AskSonic](https://github.com/srichter/asksonic#readme)
-- Other: 
+- Other:
   - [Subsonic Kodi Plugin](https://github.com/warwickh/plugin.audio.subsonic#readme)
   - [Navidrome Kodi Plugin](https://github.com/BobHasNoSoul/plugin.audio.navidrome#readme)
   - [HTTPDirFS](https://github.com/fangfufu/httpdirfs#readme)
-    
-For more options, look at the [list of clients](https://airsonic.github.io/docs/apps/) maintained by 
-the Airsonic project. Please open an [issue](https://github.com/navidrome/navidrome/issues) if you have any 
+
+For more options, look at the [list of clients](https://airsonic.github.io/docs/apps/) maintained by
+the Airsonic project. Please open an [issue](https://github.com/navidrome/navidrome/issues) if you have any
 trouble with the client of your choice.
 
 ## Road map
 
-This project is in active development. Expect a more polished experience and new features/releases 
-on a frequent basis. Some upcoming features planned: 
+This project is in active development. Expect a more polished experience and new features/releases frequently. Some upcoming features planned:
```

Comment:
I still prefer _on a frequent basis_, unless it is wrong English....

Reply:
Ok